src/encoded/docs/adding_new_assays.md

```diff
@@ -0,0 +1,283 @@
+Adding a new assay
+=========================
+
+This document describes how and where to update the encoded application to support a new assay term name.
+
+Guide to where to edit Source Code
+----------------
+
+* **src** directory - contains all the python and javascript code for front and backends
+    * **schemas** - JSON schemas ([JSONSchema], [JSON-LD]) describing allowed types and values for all metadata objects
+    * **audit** - python instructions for checking metadata stored in the schema
+    * **tests** - Unit and integration tests
+    * **types** -  business logic for dispatching URLs and producing the correct JSON
+    * **docs** -  SOPs and additional information on schemas, audits, ontologies, and annotations
+
+Steps to add a new assay
+---------------- 
+
+1. Schemas and changelogs:
+
+    Assays are identified by precise ontological terms that we list in a property *assay_term_name*. This is used in many different objects on the portal, and in most cases a mixin property ```assay``` provides this enum list. You will need to add your new assay type to the enum in mixins.json and update the changelogs of the objects that import this property.
+    The objects **Award**, **Pipeline**, and **Software** list ```assay_term_name``` in unique objects without importing the ```assay``` property, so both the schema profile and changelog will need to be updated in these cases.
```

Comment:
I think you mean "in unique properties" for those objects?

Reply:
Thank you, yes! I corrected it to 'properties'